### DISCUSSION TITLE
How do I set up OpenGraph tags for LinkedIn?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#9217] [Questions (Nuxt 2)]
I have setup the  SEO or the meta tags on each page, but sharing it in LinkedIn doesn't work

**Comment by danielroe** [comment]
@BroJenuel You might try https://pwa.nuxtjs.org/ which is designed to add the OpenGraph tags that you're likely wanting to be in place. (On the other hand, if you think there is an issue with Nuxt rendering the meta tags you have configured in `head()`, please do provide a reproduction and I'll look into it.)

**Comment by JenuelDev** [comment]
@danielroe you can check my sample blog site here https://brojenuel-blog.ml (post added are for testing only)

when I try to share it on Twitter and Facebook it works fine, it renders fine.
But when I try to share it on LinkedIn the post doesn't render the expected view.

## sample render post in twitter:
![image](https://user-images.githubusercontent.com/31676163/116552804-03a91580-a92c-11eb-8dad-96952b8ad27f.png)

## sample render post in linkedin:
![image](https://user-images.githubusercontent.com/31676163/116552951-30f5c380-a92c-11eb-9c90-79bde3c697b5.png)


I tried testing the link here in the LinkedIn URL post inspector: https://www.linkedin.com/post-inspector/

This is how I render at in each page:
```js
head() {
        return {
            title: this.article.title,
            meta: createSEOMeta({
                title: this.article.title,
                description: this.article.description,
                image: this.article.img,
                url: process.env.baseUrl +'/view/'+ this.$route.params.slug
            }),
        }
    },
```
`createSEOMeta()` this function is here:
```ts
export const createSEOMeta = (data: any) => [
    { hid: 'og:title', property: 'og:title', content: data.title },
    { hid: 'description', name: 'description', content: data.description },
    { hid: 'image', property: 'image', content: data.image },
    {
      hid: 'og:description',
      property: 'og:description',
      content: data.description,
    },
    { hid: 'og:image', property: 'og:image', content: data.image },
    {
      hid: 'og:url',
      property: 'og:url',
      content: process.env.HOST_NAME + '/' + data.url,
    },
    {
      hid: 'twitter:card',
      name: 'twitter:card',
      content: data.cardType || 'summary_large_image',
    },
]
```

You can check the source code of the website here: https://github.com/BroJenuel/BroJenuelBlog

**Reply by arikanaydin** [reply]
hello, did you find a solution?

**Reply by JenuelDev** [reply]
I just realized when I was working on this before, that it seems sharing pages for static is not really good when it comes to SEO, Also I have to use SSR for better SEO.
